.github/workflows/ci.yml

```diff
@@ -264,95 +270,3 @@ jobs:
         with:
           command: build
           args: --verbose --release
-
-  clippy-deps:
-    name: Clippy (stable)
-    timeout-minutes: 30
-    runs-on: ubuntu-latest
-    env:
-      CARGO_INCREMENTAL: 0
-      RUST_BACKTRACE: full
-
-    steps:
-      - uses: actions/checkout@v2.4.0
-        with:
-          persist-credentials: false
-
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          components: clippy
-          override: true
-
-      - uses: Swatinem/rust-cache@v1
-
-      - name: Show env vars
-        run: |
-            echo "Common env vars:"
-            echo "RUST_BACKTRACE=${{ env.RUST_BACKTRACE }}"
-            echo "Build env vars:"
-            echo "CARGO_INCREMENTAL=${{ env.CARGO_INCREMENTAL }}"
-            echo "CARGO_TARGET_DIR=${{ env.CARGO_TARGET_DIR }}"
```

Comment:
Blocker: it looks like these diagnostics were deleted.

Reply:
This one just echoes the variables available on the job to the terminal. But we have the same behavior when clicking on a specific step. We even have to go into the action and open it to see it. It's duplicating Github Action's default behavior, but manually.

![image](https://user-images.githubusercontent.com/16890942/150874798-28332805-e125-4032-9692-dbedcb6a1a6b.png)

And we don't have to download the toolchain as `actions-rs/clippy-check@v1.0.7` already does that.

Reply:
Ok, that seems fine then.